src/modules/Elsa.Studio.Workflows/Components/WorkflowInstanceViewer/Components/ActivityExecutionsTab.razor.cs

```diff
@@ -12,7 +14,7 @@ public partial class ActivityExecutionsTab
     /// Represents a row in the table of activity executions.
     /// <param name="Number">The number of executions.</param>
     /// <param name="ActivityExecution">The activity execution.</param>
```

Comment:
Let's keep the param up to date. Do we also want to update the name of the parameter?